linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost/rule.yml

```
@@ -90,3 +90,20 @@ fixtext: |-
     *.* @@[remoteloggingserver]:[port]"
 
 srg_requirement: 'The {{{ full_name }}} audit records must be off-loaded onto a different system or storage media from the system being audited.'
+
+warnings:
+    - functionality: |-
+        It is important to configure queues in case the client is sending log
+        messages to a remote server. If queues are not configured, there is a
```

Sorry, my previous suggestion did not connect well with the whole sentence.

```suggestion
        messages to a remote server. If queues are not configured,
```